routes/_components/compose/ComposeBox.html

```diff
@@ -110,58 +79,48 @@ <h1 class="sr-only">Compose status</h1>
   import ComposeLengthIndicator from './ComposeLengthIndicator.html'
   import ComposeAuthor from './ComposeAuthor.html'
   import ComposeInput from './ComposeInput.html'
-  import ComposeButton from './ComposeButton.html'
+  import ComposeStickyButton from './ComposeStickyButton.html'
   import ComposeMedia from './ComposeMedia.html'
   import ComposeContentWarning from './ComposeContentWarning.html'
   import { measureText } from '../../_utils/measureText'
   import { POST_PRIVACY_OPTIONS } from '../../_static/statuses'
   import { store } from '../../_store/store'
   import { slide } from 'svelte-transitions'
   import { postStatus, insertHandleForReply, setReplySpoiler, setReplyVisibility } from '../../_actions/compose'
-  import { importShowComposeDialog } from '../dialog/asyncDialogs'
   import { classname } from '../../_utils/classname'
-  import { observe } from 'svelte-extras'
 
   export default {
     oncreate () {
-      let { realm } = this.get()
-      if (realm === 'home') {
-        this.setupStickyObserver()
-      } else if (realm !== 'dialog') {
+      let { realm, replySpoiler, replyVisibility } = this.get()
+      if (realm !== 'home' && realm !== 'dialog') {
         // if this is a reply, populate the handle immediately
-        insertHandleForReply(realm)
+        /* no await */ insertHandleForReply(realm)
```

Comment:
Sorry, totally off topic...
I love this convention, I'm going to start doing this, too
(Kinda' wish it was an error call an asynchronous function without either `await` or `nowait`)